pkg/backend/filestate/backend.go

```diff
@@ -71,18 +84,41 @@ func (r localBackendReference) Name() tokens.QName {
 	return r.name
 }
 
-func IsLocalBackendURL(url string) bool {
-	return strings.HasPrefix(url, localBackendURLPrefix)
+func IsFileStateBackendURL(url string) bool {
+	for _, scheme := range supportedSchemes {
```

Comment:
FYI, I filed https://github.com/google/go-cloud/issues/1695 to add a function to `gocloud.dev/blob` that determines if a scheme is supported; then you could replace this with something like:

```
u, err := url.Parse(urlstr)
if err != nil {
  return false // ? or maybe true to force the error?
}
return blob.DefaultMux().IsValidScheme(u.Scheme)
```

Reply:
@Place1 looks like https://github.com/google/go-cloud/issues/1695 was merged so you could swap this out to `IsValidScheme`

Reply:
Yup -- it's only at HEAD for now but we're planning on tagging a release later this week.

Reply:
Done.  Thanks!